nishimweelysee/Elysee-CapstoneProject
# Ft user sign in page

#### What does this PR do?
>- This PR have a User Sign In Page where user have to put his/her Credentials to sign in
>
#### Description of Task to be completed
>- Email input
>- Password input
>- Sign in button
>- Other Sign in option
>
#### How should this be manually tested?
>- Open a website  navigate to sign in .
>- You will see all input fields to fill in.
>
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
> Ft(User Sign in Page)  https://trello.com/b/TKrvwje3 use this link to verify.
#### Screenshots (if appropriate)
>
![image](https://user-images.githubusercontent.com/56052588/92321730-9fe47c00-f02c-11ea-8124-faff49d03237.png)

>- Thank you !!
